lib/elements/reinforcement/straight_longitudinal_layer.rb

```diff
@@ -0,0 +1,64 @@
+require 'elements/reinforcement/base'
+require 'errors/reinforcement/invalid_distribution_direction'
+
+module Elements
+  module Reinforcement
+    class StraightLongitudinalLayer < Base
+      attr_accessor :start_location, :end_location, :rebar, :id
+
+      VALID_DIRECTIONS = %i[length_1 length_2].freeze
```

Comment:
What happens if we want to use this class later on in columns or beams? For those ones, the reinforcement direction would be in `length_3` direction, right?

Reply:
Right, so taking this into account, length_3 should be a valid direction